helm/trino-gateway/values.yaml

```diff
@@ -52,6 +52,15 @@ config:
     managedApps:
         - io.trino.gateway.ha.clustermonitor.ActiveClusterMonitor
 
+# Startup command for Trino Gateway process. Add additional java options here if required
+command:
+    - "java"
```

Comment:
two space indent like in below or four like above.. maybe we should make the whole file consistent

Reply:
Lets go with 2 - I'll make the indentation consistent in the rest of the file in a follow up PR